pyfar/dsp/interpolation.py

```diff
@@ -495,6 +496,14 @@ def resample(signal, sampling_rate, match_amplitude="auto", frac_limit=None):
         realizing the target sampling rate.
 
         The default is ``None``, which uses ``frac_limit = 1e6``.
+    suppress_aliasing : bool, optional
+        Suppress aliased energy above the Nyquist frequency of the input
+        signal, i.e., ``signal.sampling_rate/2``. This is recommended, and is
```

Comment:
Default indication missing, could e.g. be inserted here:
```suggestion
        signal, i.e., ``signal.sampling_rate/2``. This is recommended, the default is ``True``. It is
```

Reply:
Added the default.